bin/awslocal

```diff
@@ -101,19 +101,6 @@ def prepare_environment():
     env_dict = os.environ.copy()
     env_dict['PYTHONWARNINGS'] = os.environ.get(
         'PYTHONWARNINGS', 'ignore:Unverified HTTPS request')
-    if os.environ.get('DEFAULT_REGION'):
```

Comment:
I'm not quite sure what special purpose `DEFAULT_REGION` serves over `AWS_DEFAULT_REGION`, hence I've removed it